Effects/MosaicEffect/source/DemoState.hx

```diff
@@ -0,0 +1,49 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.FlxState;
+import flixel.text.FlxText;
+import flixel.tweens.FlxTween;
+import flixel.util.FlxColor;
+
+class DemoState extends FlxState
+{
+	private var isActive:Bool = false;
+	private var effect:MosaicEffect;
+	
+	override public function create():Void
+	{	
+		add(new FlxSprite(0, 0, "assets/images/backdrop.png"));
+		
+		// The effect is not a FlxObject, so no need to call the state's add()-method.
```

Comment:
It does seem a little strange that the effect adds itself to the camera's effects. Shouldn't that be done here instead?